examples/pom.xml

```diff
@@ -20,14 +20,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.apache.james</groupId>
-        <artifactId>james-project</artifactId>
-        <version>3.6.0-SNAPSHOT</version>
-    </parent>
+    <groupId>org.apache.examples.james</groupId>
```

Comment:
I'm unsure that the James project is allowed to publish JARs to maven central  with a `org.apache.examples.` prefix. 

Can you check the Apache foundation policies regarding this?

Reply:
https://infra.apache.org/publishing-maven-artifacts.html don't seem helpful regarding this concern...

Reply:
Neither do https://infra.apache.org/release-distribution.html ...

Reply:
Maybe we need to ask the INFRA team?

Reply:
Also using org.apache.examples prefix don't seem to be a standard practice amongst other project.

See https://mvnrepository.com/search?q=org.apache.examples

Maybe we should keep the `org.apache.james.examples` FQDN while switching it to a top level POM not depending on James prent POM. ?

Reply:
After thinking about this I realised that if we have all example code under org.apache.james.examples then we cannot use package level access anywhere (as core james won't have the 'examples' part in the package name). Sorry for creating confusion - reverted back this part - let's stay to org.apache.james.examples